### ISSUE TITLE
Connect with Service Design for VFS-need-to-know items

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#62209]
## User Story
As a new VFS team member, I want to be introduced to need-to-know information so that I'm not spending my time reviewing documents and articles that are not relevant and/or helpful.

Assignee: @shiragoodman 
Peer Reviewer:

## Description
Service Design team has completed a lot of research wrt what VFS team members need to know when building on the VA.gov Platform. In an effort to streamline the tasks asked of VFS team members within Platform Orientation, Governance will be meeting with Service Design to help determine what tasks should be removed from orientation and what tasks should be added.

In addition, there are Service Design-owned products currently referenced in the Platform Orientation ticket. This ticket will also identify what Service Design-owned products/services should be included in the ticket

## Impacted Artifacts
[New VFS Team Member gh template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=analytics-insights%2C+analytics-request%2C+governance-team%2C+platform-orientation&projects=&template=orientation-epic.md&title=Platform+Orientation+Template+%5BYour+name+here%5D)

## Tasks
- [x] Shira to sync with Bryan

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] Governance has synched with Service Design to determine VFS need-to-know items

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@shiragoodman synched with @steel36 on Tuesday 8/8. 

At the Platform Huddle on 8/15, it was proposed that Governance put this initiative on hold until additional discovery can be completed by Service Design. Shira will discuss this with @humancompanion-usds on 8/18 at their 1:1 sync

**Comment by shiragoodman** [comment]
@steel36 I know your team had done some initial discovery on Platform Orientaiton. Would you mind sharing the link here? Is your team planning to do additional research?

**Comment by shiragoodman** [comment]
thanks @steel36 ! I'm asking for the same exact reason - roadmap and prioritization for 2024. I'll plan to bring this up with Matt and will follow up with you once we know where this lands.
